client/app/components/queries/EmbedQueryDialog.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import Modal from 'antd/lib/modal';
+import Typography from 'antd/lib/typography';
+import { wrap as wrapDialog, DialogPropType } from '@/components/DialogWrapper';
+import { clientConfig } from '@/services/auth';
+
+const { Text } = Typography;
+
+function EmbedQueryDialog({ dialog, query, visualization }) {
+  const embedUrl = `${clientConfig.basePath}embed/query/${query.id}/visualization/${
+    visualization.id}?api_key=${query.api_key}&${query.getParameters().toUrlParams()}`;
+
+  let snapshotUrl;
+  if (window.snapshotUrlBuilder) {
```

Comment:
I'm not sure what this is for, but I just followed what was in the Angular version.